components/brave_vpn/browser/brave_vpn_service_unittest.cc

```diff
@@ -125,8 +125,9 @@ std::string GenerateTestingCreds(const std::string& domain,
   auto now = base::Time::Now();
   base::Time::Exploded exploded;
   now.LocalExplode(&exploded);
+  // Give a sufficient additional yer to prevent it expired.
```

Comment:
// Give sufficient additional years to prevent it expired.

Reply:
Updated. :thanks!